dockerfile

```diff
@@ -0,0 +1,29 @@
+ARG PYTHON_VERSION=3.9
+
+FROM python:${PYTHON_VERSION} as builder
+
+WORKDIR /app
+
+ENV PYTHONUNBUFFERED=1
+
+COPY . /app
+
+RUN pip install --upgrade pip && pip install -r requirements.txt
+
+RUN python manage.py migrate 
+
```

Comment:
The step with the installation of packages can be removed and the step with migration can be moved in run stage